doc/classes/BaseMaterial3D.xml

```diff
@@ -393,6 +393,10 @@
 		<member name="texture_repeat" type="bool" setter="set_flag" getter="get_flag" default="true">
 			Repeat flags for the texture. See [enum TextureFilter] for options.
 		</member>
+		<member name="texture_smooth_pixel_filter" type="bool" setter="set_flag" getter="get_flag" default="false">
+			If [code]true[/code], displays the texture with an appearance that mimics nearest-neighbor filtering with a shader to smooth out aliasing between pixels. This can be used for a pixel art look that is still antialiased regardless of the current antialiasing method in use.
+			[b]Note:[/b] This internally forces the texture filter mode to the corresponding Linear filter mode, as the smooth pixel filtering shader relies on linear filtering to work.
```

Comment:
May be worth accentuating since it's also talking about the enum constants.
```suggestion
			[b]Note:[/b] This internally forces the texture filter mode to the corresponding [b]Linear[/b] filter mode, as the smooth pixel filtering shader relies on linear filtering to work.
```